src/quic/node_quic_util-inl.h

```diff
@@ -424,14 +424,6 @@ void StatsBase<T>::StatsMemoryInfo(MemoryTracker* tracker) const {
   tracker->TrackField("ack_histogram", ack_);
 }
 
-template <typename T>
-StatsBase<T>::~StatsBase() {
-  StatsBase<T>::StatsDebug stats_debug(static_cast<typename T::Base*>(this));
-  Debug(static_cast<typename T::Base*>(this),
-        "Destroyed. %s",
-        stats_debug.ToString().c_str());
```

Comment:
It might be nice to keep this as a function on `StatsBase` that is called from the individual destructors, rather than end up duplicating the code in each individual one?

Reply:
Ah right, can give that a try

Reply:
Works! Updated